database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/UserViewModel.swift

```diff
@@ -30,6 +30,10 @@ class UserViewModel: ObservableObject {
     Database.database().reference()
   }()
 
+  enum PostListType {
```

Comment:
Can we move the definition to a separate file or to the line 19 of this file to be in the global scope? It doesn't look like it belongs to `UserViewModel` exclusively.